kancolle_auto.sikuli/combat.sikuli/combat.py

```diff
@@ -10,7 +10,7 @@
 # Custom similarity thresholds
 DMG_SIMILARITY = 0.7  # Damage state icons
 FATIGUE_SIMILARITY = 0.98  # Fatigue state icons
-CLASS_SIMILARITY = 0.7  # Ship class icons
+CLASS_SIMILARITY = 0.76  # Ship class icons
```

Comment:
More of a note to self: it seems like `CLASS_SIMILARITY` might be in flux considering the other PR.